lib/npg_tracking/glossary/composition/component/illumina.pm

```diff
@@ -12,6 +12,16 @@ with qw( npg_tracking::glossary::composition::component
 
 our $VERSION = '0';
 
+sub filename {
+  my ($self, $ext) = @_;
+  $ext //= q[];
```

Comment:
> Do we need a minumum Perl version of 5.10 (Build.PL or code?) now 
> we're using this operator?
>
Requested at least perl 5.10 in Build.PL.




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.